### ISSUE TITLE
Bug(?): Mage's Disjunction and Selective Meta Magic

### ISSUE THREAD

**Issue by TKDancer** [Vek17/TabletopTweaks-Base#645]
another user reported it as a bug that it was previously being able to use selective MM and has seemingly since been 'fixed'?(tho they posted that on the day of the latest release, april 24th, so i'm not sure)

however i can't apply the MM to it myself, this despite the spell clearly being an instantaneous one that then tries to apply a lingering effect to the gear of affected targets(rather than creating a continuous bubble than suppresses magical effects of magic items like some huge anti-magic field) so it seems to be that RAI it should be able to use Selective

**Comment by CaptOcto** [comment]
Yeah I agree that spells like this (which also includes Confusion) should be able to have Selective applied. However on the flipside I think MD is too strong and needs to be toned down (caster level check vs each affect, 5% chance each item is Disjunction for 1 day/level).
